osi_hostvehicledata.proto

```diff
@@ -32,6 +32,14 @@ message HostVehicleData
     //
     optional Timestamp timestamp = 11;
 
+    // The ID of the object.
```

Comment:
I did think about this but struggled since you can now send this message independently of any GroundTruth data.  Nonetheless, I agree that being specific is valuable so I have still referred to GroundTruth given that is likely what will determine the ID.